v22.1/cockroach-cert.md

```diff
@@ -182,6 +105,73 @@ Note the following:
 
 - This check can be disabled by setting the environment variable `COCKROACH_SKIP_KEY_PERMISSION_CHECK` to `true`.
 
+## Synopsis
+
+Create the CA certificate and key:
+
+~~~ shell
+$ cockroach cert create-ca \
```

Comment:
I think we use curly braces for replaceable text. https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#placeholders